config/initializers/omniauth.rb

```diff
@@ -0,0 +1,24 @@
+OmniAuth.config.logger = Rails.logger
+
+Rails.application.config.middleware.use OmniAuth::Strategies::CognitoIdP,
+    Rails.application.secrets.cognito_client_id,
+    Rails.application.secrets.cognito_client_secret,
+    scope: 'email openid aws.cognito.signin.user.admin profile',
+    aws_region: Rails.application.secrets.cognito_aws_region,
+    user_pool_id: Rails.application.secrets.cognito_user_pool_id,
+    client_options: {
+        site: Rails.application.secrets.cognito_user_pool_site
+    }
+
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :developer, :fields => [:name, :email, :phone, :first_name, :last_name],
+  :uid_field => :last_name unless Rails.env.production?
```

Comment:
This line is very confusing for me. Could we wrap the whole builder in the `Rails.env.production?` conditional?